doc/source/whatsnew/v1.5.0.rst

```diff
@@ -245,6 +245,14 @@ and attributes without holding entire tree in memory (:issue:`45442`).
 .. _`lxml's iterparse`: https://lxml.de/3.2/parsing.html#iterparse-and-iterwalk
 .. _`etree's iterparse`: https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.iterparse
 
+.. _whatsnew_150.enhancements.pandas-stubs:
+
+``pandas-stubs``
+^^^^^^^^^^^^^^^^
+
+The ``pandas-stubs`` library is now supported by the pandas development team, providing type stubs for the pandas API. Please visit
+https://github.com/pandas-dev/pandas-stubs for more information.
```

Comment:
There is also a reference to pandas-stubs in `doc/source/ecosystem.rst` referring to VirtusLab. should probably either remove that section or update in this PR also.

Reply:
should maybe also give acknowledgment to VirtusLab and Microsoft

Reply:
I think `doc/source/ecosystem.rst` was updated in https://github.com/pandas-dev/pandas/pull/47869